client/src/components/DirectMessage/Chat.jsx

```diff
@@ -0,0 +1,47 @@
+import axios from 'axios';
+import React, {useState, useEffect, useContext} from 'react';
+import { async } from 'regenerator-runtime';
+
+
+const ChatOnline = ({conversation, currentUser}) => {
+
+  //hold state of user dealing with the conversation involved
+  const [user, setUser] = useState('ben');
```

Comment:
you have this hardcoded with 'ben'... maybe set the default state to an empty string if you dont want 'ben' hardcoded